### ISSUE TITLE
Unknown OpenFlow traffic and missing packets when following Mininet walkthrough

### ISSUE THREAD

**Issue by Subject22** [mininet/mininet#622]
## Issue 1: Unexpected OpenFlow traffic
The Mininet walkthrough says that after starting up Wireshark and Mininet that "for now, there should be no OpenFlow packets displayed in the main window." However, as shown in the attached screenshot, I see OpenFlow packets on the loopback interface every few seconds (`ECHO_REQUEST` and `ECHO_REPLY`). Is this expected behaviour? Am I doing something incorrectly? This may simply be a case of me not understanding something, as opposed to a problem with Mininet.

## Issue 2: Missing ping packets
I also note that while running `h1 ping -c 1 h2` twice (after creating a minimal topology with `sudo mn`) generates the expected output in the command line (ie: two successful pings, with the first having a longer RTT than the second) the expected packets do not appear in Wireshark (ie: the  `PACKET_IN`s and `PACKET_OUT`s are missing). The packets expected as a result of the first ping appear, but the packets expected as a result of the second do not. After some trial and error I have found that waiting for a couple of minutes between pings results in the expected packet activity always showing up in Wireshark, but that when performing pings in quick succession I see no `PACKET_IN`s or `PACKET_OUT`s for the latter pings.

## Environment Setup/Background
As you can see from the attached screenshot I am running Mininet in the preconfigured VM and piping the output of Dumpcap (running on the VM) to Wireshark on my host machine via SSH. This is slightly different to the setup recommended in the Mininet VM setup notes, but I would not have expected this to cause problems. Have I missed something?

![fullsizerender](https://cloud.githubusercontent.com/assets/4090877/14803045/53c2e034-0baa-11e6-885d-0e94bd234bec.jpg)

**Comment by lantz** [comment]
(2) should be expected behavior if you understand OpenFlow.
(1) is also expected - perhaps we should clarify the walkthrough.

**Comment by Subject22** [comment]
Yes, my understanding of OpenFlow has improved substantially since I posted this issue (hence why I closed it) 😉 

Clarifying the walkthrough would probably have helped me at the time. Let me know if you'd like any further feedback.
